libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef.dat

```diff
@@ -294,8 +294,8 @@ PE10 SAFETY_IN INPUT PULLDOWN
 # enable FAT filesystem support (needs a microSD defined via SDMMC)
 define HAL_OS_FATFS_IO 1
 
-define HAL_BOARD_LOG_DIRECTORY "/APM/LOGS"
-define HAL_BOARD_TERRAIN_DIRECTORY "/APM/TERRAIN"
+define HAL_BOARD_LOG_DIRECTORY HAL_BOARD_STORAGE_DIRECTORY "/LOGS"
+define HAL_BOARD_TERRAIN_DIRECTORY HAL_BOARD_STORAGE_DIRECTORY "/TERRAIN"
```

Comment:
This identical patch exists in *how* many hwdef.dat files?!

Something is wrong here.  Perhaps this should be added into the hwdef.py script instead - or just removed after verifying that this is the default value anyway!

Reply:
I installed it on my CONTROL ZERO CLASSIC.
I am able to confirm that this change is effective.

![Screenshot from 2021-09-19 11-42-20](https://user-images.githubusercontent.com/646194/133913670-d8facfcc-ce0b-40d9-82dc-e5e5c021eab1.png)

![Screenshot from 2021-09-19 11-42-36](https://user-images.githubusercontent.com/646194/133913676-e838eb14-68fd-4791-a073-445701ea36dd.png)